### PR TITLE
adding axioms to obtain same subclass-inference for male and female reproductive system gland secretion

### PR DESCRIPTION
**Pull request by drseb**
related to https://github.com/obophenotype/uberon/issues/1619

### CODE REVIEW

File: uberon_edit.obo [hunk 1]
```diff
@@ -2268,6 +2268,7 @@ xref: VHOG:0000725
 xref: WBbt:0008423
 xref: Wikipedia:Male_reproductive_system_(human)
 xref: XAO:0000155
+is_a: UBERON:0000990 ! reproductive system
```
Comment (matentzn) [inline uberon_edit.obo:2271]: Is this the only change? The rest of the diff seems just spurious (I checked it all, nothing broke, just wanna be sure).

Comment (drseb) [reply uberon_edit.obo:2271]: sorry. I just saw another inconsistency. so two changes are introduced in this branch.

Comment (cmungall) [reply uberon_edit.obo:2271]: also this shouldn't be necessary as we have the equiv axiom

Comment (drseb) [reply uberon_edit.obo:2271]: ok. but the other axiom is necessary? I think the first change makes the female and male axioms consistent, right?

Comment (drseb) [reply uberon_edit.obo:2271]: should I make a new PR?

**Pull request closed without merging**